Kernel/ACPI/Parser.h

```diff
@@ -45,7 +45,7 @@ class Parser {
     template<typename ParserType>
     static void initialize(PhysicalAddress rsdp)
     {
-        set_the(*new ParserType(rsdp));
+        set_the(*new (nothrow) ParserType(rsdp));
```

Comment:
No point in making this nothrow if we immediately dereference the pointer?

Reply:
Yes, you're right. However, I plan to try to merge this after #9934 is merged.